src/server/index.js

```diff
@@ -58,8 +72,10 @@ function Server({ games, db }) {
           const state = store.getState();
 
           // Get clients connected to this current game.
-          const roomClients = roomInfo.get(gameID);
-          for (const client of roomClients.values()) {
+          const roomClients = Array.from(roomInfo.get(gameID))
+            .map(playerID => playerInfo.get(playerID))
+            .filter(socketID => socketID !== undefined);
```

Comment:
Checking for `undefined` is probably unnecessary after https://github.com/google/boardgame.io/commit/a768f1f8a8775360b354f8475da77b96ca029535

Reply:
I will look at that commit more, but my quick glance confirms this check is now unnecessary.